Google.Api.Gax.Grpc/GaxGrpcServiceCollectionExtensions.cs

```diff
@@ -21,15 +21,19 @@ public static class GaxGrpcServiceCollectionExtensions
         /// Adds a singleton <see cref="GrpcNetClientAdapter"/> to the given service collection
         /// as the preferred <see cref="GrpcAdapter"/> implementation,
         /// using the default instance with any additional options configured via <paramref name="optionsConfigurer"/>.
+        /// Before executing the specified action, 
```

Comment:
This seems incomplete.

Reply:
Whoops, thanks - will fix the comment then merge.